src/Google/GlobalSiteTag.php

```diff
@@ -355,6 +356,63 @@ public function maybe_display_conversion_and_purchase_event_snippets( string $ad
 			join( ',', $item_info ),
 		);
 		wp_print_inline_script_tag( $purchase_page_gtag );
+
+		// Enhanced conversion data.
+		$ec_data         = [];
+		$customer        = new WC_Customer( $order->get_customer_id() );
+		$email           = $customer->get_billing_email();
+		$fname           = $customer->get_billing_first_name();
+		$lname           = $customer->get_billing_last_name();
+		$phone           = $customer->get_billing_phone();
+		$billing_address = $customer->get_billing_address();
+		$postcode        = $customer->get_billing_postcode();
+		$city            = $customer->get_billing_city();
+		$region          = $customer->get_billing_state();
+		$country         = $customer->get_billing_country();
+
+		// Add email in EC data.
+		if ( ! empty( $email ) ) {
+			$ec_data['email'] = $email;
+		}
+
+		// Format phone number in IE64.
+		$phone        = preg_replace( '/[^0-9]/', '', $phone );
+		$phone_length = strlen( $phone );
+		if ( $phone_length > 9 && $phone_length < 14 ) {
+			$phone            = sprintf( '%s%d', '+', $phone );
+			$ec_data['email'] = $email;
```

Comment:
I think this was meant to be where you set ec_data for phone:

```suggestion
			$ec_data['phone_number'] = $phone;
```